docs/gl_objects/groups.rst

```diff
@@ -226,11 +226,15 @@ Reference
   + :class:`gitlab.v4.objects.GroupMember`
   + :class:`gitlab.v4.objects.GroupMemberManager`
   + :class:`gitlab.v4.objects.GroupMemberAllManager`
+  + :class:`gitlab.v4.objects.GroupBillableMember`
+  + :class:`gitlab.v4.objects.GroupBillableMemberManager`
   + :attr:`gitlab.v4.objects.Group.members`
   + :attr:`gitlab.v4.objects.Group.members_all`
+  + :attr:`gitlab.v4.objects.Group.billable_members`
 
-* GitLab API: https://docs.gitlab.com/ce/api/groups.html
+* GitLab API: https://docs.gitlab.com/ce/api/members.html
 
+Billable group members are only available in GitLab EE.
```

Comment:
Yes, technically gitlab.com is running an EE instance. Right now at http://gitlab.com/help I get `GitLab Enterprise Edition 13.12.0-pre 9327bd41b84 `. But if you don't pay it's like without a license. Same with any instance running the `gitlab/gitlab-ee` image, it supports a few extra endpoints even if you don't have a license entered. So in this case you can test billable_members on gitlab.com yourself, on any top-level group now (I did already).

But I didn't think it was worth adding an extra functional test job with a separate container because the endpoints are very limited without a license. For example I noticed that I can use EE/gitlab.com to test some Premium features like MR approval rules (even though I can't add new ones), but it just returns an empty array whereas with CE it returns I think 403 or 404.

Reply:
Thanks!